web/src/utils/language.ts

```diff
@@ -0,0 +1,7 @@
+import { useTranslation } from 'react-i18next';
+
+export function currentLanguage(): string {
+  const { i18n } = useTranslation();
```

Comment:
You should not use a hook outside of react component or other hook. If you need to get i18n object, you can get it from `i18next`, not `i18next-react`.